relationships-api/src/main/java/org/commonjava/maven/atlas/graph/model/EProjectDirectRelationships.java

```diff
@@ -42,25 +53,27 @@
 
     private static final long serialVersionUID = 1L;
 
-    private final URI source;
+    private URI source;
 
-    private final ProjectVersionRef ref;
+    private ProjectVersionRef projectRef;
 
-    private final List<BomRelationship> boms;
+    private List<BomRelationship> boms;
 
-    private final List<DependencyRelationship> dependencies;
+    private List<DependencyRelationship> dependencies;
 
-    private final List<DependencyRelationship> managedDependencies;
+    private List<DependencyRelationship> managedDependencies;
 
-    private final List<PluginRelationship> plugins;
+    private List<PluginRelationship> plugins;
 
-    private final List<PluginRelationship> managedPlugins;
+    private List<PluginRelationship> managedPlugins;
 
-    private final List<ExtensionRelationship> extensions;
+    private List<ExtensionRelationship> extensions;
 
-    private final ParentRelationship parent;
+    private ParentRelationship parent;
 
-    private final Map<PluginRelationship, List<PluginDependencyRelationship>> pluginDependencies;
+    private Map<PluginRelationship, List<PluginDependencyRelationship>> pluginDependencies;
```

Comment:
Sounds great. I did not know why PluginRelationship being as a map key but that feels too heavy. I will do as you mentioned, and fix any incompatible code due to this api change.